From: Raul Rodrigo <raurodse@gmail.com>
Date: Wed, 2 Sep 2020 19:54:45 +0200
Subject: another patch

---
 germinate/germinator.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/germinate/germinator.py b/germinate/germinator.py
index be55eb0..3f6080d 100644
--- a/germinate/germinator.py
+++ b/germinate/germinator.py
@@ -847,7 +847,7 @@ class Germinator(object):
                     if posarch and self._arch not in posarch:
                         continue
 
-            #pkg = pkg.split()[0]
+            # pkg = pkg.split()[0]
             try:
                 pkg = pkg.split()[0]
             except Exception as e:
